stdlib/LinearAlgebra/src/diagonal.jl

```diff
@@ -1036,6 +1036,16 @@ end
 *(A::AbstractTriangular, D::Diagonal) = @invoke *(A::AbstractMatrix, D::Diagonal)
 *(D::Diagonal, A::AbstractTriangular) = @invoke *(D::Diagonal, A::AbstractMatrix)
 
+_opnorm12Inf(A::Diagonal, p) = maximum(opnorm(x, p) for x in A.diag)
+
+function opnorm(A::Diagonal, p::Real=2)
+    if p == 1 || p == Inf || p == 2
+        return _opnorm12Inf(A, p)
+    else
+        throw(ArgumentError("invalid p-norm p=$p. Valid: 1, 2, Inf"))
+    end
+end 
```

Comment:
```suggestion
end
```

Reply:
@dgleich would you be able to accept this change? I can't since the PR is from your fork of Julia.

Reply:
Is it just the one line ending? If so, I pushed a change.

Reply:
Yes. It was just the white space here.